pm_muairss_write/pm_muairss_write.xml

```diff
@@ -3,27 +3,43 @@
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
         pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
-        cp $structure input.cell &&
-        cp $params params.yaml &&
+        ln -s $structure input.cell &&
+        ln -s $params params.yaml &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            ln -s $castep_param castep.param &&
+        #end if
         pm-muairss -t w input.cell params.yaml &&
-        zip -r out_zip.zip $out_folder &&
-        find $out_folder > tree.txt
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            zip -r out_zip1.zip \$out_folder params.yaml input.cell castep.param &&
```

Comment:
It'd be nice to use the original input file names here if we can to avoid confusion if people unzip the files, but I'm undecided if this is truly necessary. What do you think?